exasol_sagemaker_extension/deployment/deploy_cli.py

```diff
@@ -19,15 +19,25 @@ def main(args):
     parser.add_argument("--schema", help="schema name", required=True)
     parser.add_argument("--print", help="print out statements",
                         required=False, action="store_true")
+    parser.add_argument("--develop",  help="generate and execute the scripts",
```

Comment:
hm, I think, a refactoring for the future, could be using click for the CLI, maybe create refactoring Github issue for it